.github/workflows/build_deploy.yml

```diff
@@ -69,6 +71,7 @@ jobs:
           CIBW_TEST_COMMAND: python {project}/tests/smoke_test.py
           # Testing arm on MacOS is currently not supported by Github
           CIBW_TEST_SKIP: "*-macosx_universal2:arm64"
+          CIBW_ENVIRONMENT_MACOS: SYSTEM_VERSION_COMPAT=0
```

Comment:
What does this do?

Reply:
This allows to install the macos-11.0-compatible-wheel in the macos 11.0 runner instance, as the macos version numbering is a bit messy from Big Sur release apparently. You can read more here:
https://eclecticlight.co/2020/08/13/macos-version-numbering-isnt-so-simple/
https://github.com/actions/runner-images/discussions/6099

Reply:
comment would be good, I'll forget this in 2 weeks :)